README.md

```diff
@@ -21,35 +30,47 @@
 - _for i in ./deploy/controller-configs/*yaml; do kubectl apply -f $i; done_
 
-> ###### Deploy sample custom resources on the kubernetes cluster
+> ##### Deploy sample custom resources on the kubernetes cluster (Optional)
 - _for i in ./deploy/sample-crs/*yaml; do kubectl apply -f $i; done_
 
 > ##### Deploying an API in K8s cluter
 
 - Download sample API definition (swagger files) from [product micro-gateway sample](https://github.com/wso2/product-microgateway/tree/master/samples) github location.
 - Execute the following command to deploy the API
-- *kubectl add api "api_name" --from-file="location to the api swagger definition"*
-- ex: ***kubectl add api petstorebasic --from-file=petstore_basic.yaml***
-> ###### Undeploy the changes (one by one)
-
-- *kubectl delete -f deploy/crds/wso2_v1alpha1_targetendpoint_cr.yaml*
-- *kubectl delete -f deploy/crds/wso2_v1alpha1_security_cr.yaml*
-- *kubectl delete -f deploy/crds/wso2_v1alpha1_ratelimiting_cr.yaml*
-- *kubectl delete -f deploy/crds/wso2_v1alpha1_api_cr.yaml*
-- *kubectl delete -f deploy/operator.yaml*
-- *kubectl delete -f deploy/role.yaml*
-- *kubectl delete -f deploy/role_binding.yaml*
-- *kubectl delete -f deploy/service_account.yaml*
-- *kubectl delete -f deploy/crds/app_v1alpha1_appservice_crd.yaml*
-- *kubectl delete -f deploy/crds/wso2_v1alpha1_targetendpoint_crd.yaml*
-- *kubectl delete -f deploy/crds/wso2_v1alpha1_security_crd.yaml*
-- *kubectl delete -f deploy/crds/wso2_v1alpha1_ratelimiting_crd.yaml*
-- *kubectl delete -f deploy/crds/wso2_v1alpha1_api_crd.yaml*
-- *kubectl delete -f deploy/namespace.yaml* 
-- *kubectl delete -f deploy/controller_conf.yaml*
-- *kubectl delete -f deploy/docker_secret_template.yaml*
-- *kubectl delete -f deploy/analytics_secret_template.yaml*
+    - *kubectl add api "api_name" --from-file="location to the api swagger definition" --replicas="number of replicas" -n="desired namespace"*
+    - ex: ***kubectl add api petstorebasic --from-file=petstore_basic.yaml --replicas=2 -n=wso2***
+
+- Execute the following command to update the API
+    - *kubectl update api "api_name" --from-file="location to the api swagger definition" --replicas="number of replicas" -n="desired namespace"*
+    - ex: ***kubectl update api petstorebasic --from-file=petstore_basic.yaml --replicas=1 -n=wso2***
+
+- Execute the following command to view the details of the API
+    - *kubectl show api "api_name" -n="desired namespace"*
+    - ex: ***kubectl show api petstorebasic -n=wso2***
+
+- Execute the following command to remove the API
+    - *kubectl delete api "api_name" -n="desired namespace"*
+    - ex: ***kubectl delete api petstorebasic -n=wso2***
+
+- Execute the following command to remove the apim operator
+    - ***kubectl delete deployment apim-operator -n=wso2-system***
+    
+
+- Note:
+> If the namespace is not provided, default namespace will be used. The namespace used in all the commands must match.
```

Comment:
Shall we mention both namespace and replicas are optional? And if the user does not provide these, default values will be set.